classes/migration/install/CommonMigration.inc.php

```diff
@@ -95,14 +95,13 @@ public function up(): void
 
         // Locale-specific user data
         Schema::create('user_settings', function (Blueprint $table) {
+            $table->bigInteger('user_settings_id')->autoIncrement();
```

Comment:
Added for the sake of normalization.

Reply:
There are a lot of `_settings` tables and none of them currently have IDs; if we're going to start introducing them I'd like to consider it in its own issue, and if it seems promising, make the change all at once.

Reply:
Ok! I've opened an issue about normalizing the database, so I'll create a PR there.